frontend/containers/account-type-selector/account-type-item/component.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { usePress, useFocusWithin } from '@react-aria/interactions';
+import noop from 'lodash-es';
+
+import { AccountTypeItemProps } from './types';
+
+export const AccountTypeItem: FC<AccountTypeItemProps> = ({
+  accountTypeId,
+  name,
+  description,
+  imageSrc,
+  imageTitle,
+  onClick = noop,
+}: AccountTypeItemProps) => {
+  const [isFocusWithin, setIsFocusWithin] = useState<boolean>(false);
+
+  const intl = useIntl();
+
+  const { pressProps } = usePress({
+    onPress: () => onClick(accountTypeId),
+  });
+
+  const { focusWithinProps } = useFocusWithin({
+    onFocusWithinChange: setIsFocusWithin,
+  });
+
+  return (
+    <div
+      className={cx({
+        'flex flex-col items-center gap-3 px-12 py-8 text-center bg-white border rounded-xl': true,
+        'cursor-pointer hover:ring-1 hover:ring-green-dark transition-all': true,
+        'ring-2 ring-green-dark': isFocusWithin,
```

Comment:
I don't think you need `useFocusWithin`:
```suggestion
        'focus-within:ring-2 ring-green-dark': true,
```